### ISSUE TITLE
DAML Assistant quickstart-java is broken, refers to nonexistent iou.dar

### ISSUE THREAD

**Issue by leo-da** [digital-asset/daml#742]
Steps to reproduce:
```
$ bazel build //...
...

$ daml-sdk-head
...

$ daml-head new java-codegen-test
...

$ cd java-codegen-test

$ daml-head package
Created dist/java-codegen-test.dar.

$ cat ./pom.xml |fgrep -C5 "iou.dar"
                            <arguments>
                                <argument>-o</argument>
                                <argument>${daml-codegen-java.output}</argument>
                                <argument>-d</argument>
                                <argument>com.digitalasset.quickstart.iou.TemplateDecoder</argument>
                                <argument>${project.build.directory}/daml/iou.dar=com.digitalasset.quickstart.model</argument>
                            </arguments>
                            <sourceRoot>${daml-codegen-java.output}</sourceRoot>
                        </configuration>
                    </execution>
                    <execution>
```
`iou.dar` does not exist, it is `dist/java-codegen-test.dar`
This is due to the fact that the DAR file is named after the project name. See: #652

cc @nicholassmith-da @stefanobaghino-da @gerolf-da

**Comment by cocreature** [comment]
The whole documentation hasn’t been updated for the new assistant so far (it’s not the default atm). See #720 and #740. So it probably makes sense to close this issue.

**Comment by leo-da** [comment]
it is not for the documentation BTW... it is the quickstart-java template that is part of the DAML assistant that is what is broken

**Comment by cocreature** [comment]
It is the documentation that is broken. If you run `daml build -o target/daml/iou.dar` (which is what #720  uses) instead of `daml package` it works just fine.

**Comment by leo-da** [comment]
this issue should be addressed as part of #720
